recipe/meta.yaml

```diff
@@ -1,52 +1,43 @@
 {% set name = "category_encoders" %}
-{% set version = "2.6.1" %}
+{% set version = "1.3.0" %}
```

Comment:
You are trying to update to the version that is lower than we have on defaults. You should open the PR against the branch `master-1.3.0` but not `master`. See docs https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Creating_versioned_feedstocks_from_existing_packages.md

Reply:
Thanks Serhii for catching this!